src/api-reference/expense/travelallowance/travelallowance-days.md

```diff
@@ -0,0 +1,192 @@
+---
+title: TravelAllowance Days
+layout: reference
+---
+
+This Travel Allowance API fetches all the fixed allowance days in an expense report. Clients using this API can determine whether an allowance is a full day or partial day allowance.
+
+> **Limitations**: This API is available only for Finland customers.
+
+* [Process Flow](#process-flow)
+* [Products and Editions](#products-editions)
+* [Scope Usage](#scope-usage)
+* [Access Token Usage](#access-token-usage)
+* [Get Details of TravelAllowance Days](#details-of-travelallowance-days)
+* [Schema](#schema)
+  * [Error](#schema-error)
+* [Definitions](#definitions)
+
+## <a name="process-flow"></a>Process Flow
+
+![Image of Process_Flow](Process_Flow.png)
+
+
+## <a name="products-editions"></a>Products and Editions
+
+* Concur Expense Professional Edition
+* Concur Expense Standard Edition
+
+## <a name="scope-usage"></a>Scope Usage
+
+Name|Description|Endpoint
+---|---|---|
+travelallowance.allowancedays.read|View the allowance days in an expense report|GET
+
+## <a name="access-token-usage"></a>Access Token Usage
+
+This API will work with Company access token only. A valid Company `access token` is needed to use this API.
+
+## Retrieve a Company Access Token
+
+Using Company-level access token is required for creating an integration with this API. This method will allow your integration to get fixed allowance day details of an expense report.  
+
+For clients connecting to this API to build a custom integration you will receive client credentials and information on how to generate your company access token or company refresh token from the Web services resource assigned to assist you with Authentication.
+
+Before making requests to this API, you must [obtain an access token from the Authentication API](/api-reference/authentication/getting-started.html).
+
+The response will include an `access_token` field, which contains your access token. For subsequent calls, you will need to include this access token in the `Authorization` header of your calls. You will need this in order to get fixed allowance day details.
+
+### Scopes
+
+`travelallowance.allowancedays.read` - Refer to [Scope Usage](#scope-usage) for full details.
+
+### Request
+
+#### URI
+
+##### Template
+
+```shell
+https://{datacenterURI}/api/v4/travelallowance/allowancedays/companyUUID/{companyUUID}/contexts/
+{Context}/{ContextId}
+```
+
+##### Parameters
+
+Name|Type|Format|Description
+---|---|---|---
+`companyUUID`|`uuid`|-|**Required** Note required when applicable.
+`Context`|`string`|-|This parameter is required. For expense reports Context is `EXPENSE_REPORT`
+`ContextId`|`string`|-|This parameter is required. For expense reports reportId of the report needs to be passed as contextId
```

Comment:
Where do they get the report ID from? Do they need to call the Expense Report API?

Reply:
They will call v3 API's to get the report ID.
https://developer.concur.com/api-reference/expense/expense-report/reports.html